.gitignore

```diff
@@ -31,7 +31,7 @@ request-management-api/request_api/logs/*
 request-management-api/request_api/__pycache__/__init__.cpython-38.pyc
 
 foi-web/forms-flow-ai/forms-flow-web/node_modules
-foi-web/forms-flow-ai/forms-flow-web/package-lock.json
+
```

Comment:
Great!  We want a package-lock.json for OpenShift, it helps with reproducible builds.